frontend/src/store/slices/accessRequests/useAccessRequests.ts

```diff
@@ -0,0 +1,166 @@
+import { useAppDispatch } from './../../hooks';
+import { request, success, error } from 'actions/genericActions';
+import { showLoading, hideLoading } from 'react-redux-loading-bar';
+import CustomAxios from 'customAxios';
+import { ENVIRONMENT } from 'constants/environment';
+import { AxiosResponse, AxiosError } from 'axios';
+import { IAccessRequest } from 'interfaces';
+import * as actionTypes from 'constants/actionTypes';
+import * as API from 'constants/API';
+import React from 'react';
+import {
+  updateAccessRequestsAdmin,
+  storeAccessRequests,
+  deleteAccessRequest,
+  storeAccessRequest,
+} from './accessRequestsSlice';
+
+export const useAccessRequests = () => {
+  const dispatch = useAppDispatch();
+  /**
+   * Get the fetchCurrent function with returns the current access request for the current user.
+   * @returns The dispatchable action which will return the access request if one exists, or 204 if one doesn't
+   */
+  const fetchCurrent = React.useCallback(async (): Promise<IAccessRequest> => {
+    dispatch(request(actionTypes.GET_REQUEST_ACCESS));
```

Comment:
Note that these dispatches are still calling the old network reducer and so appear in the old format still. Will need to update this when that reducer is converted to the new format.

Reply:
I suspect this answers my questions above about removing the extra string file.  It's in progress.